test/integration/node-specific/auto_connect.test.ts

```diff
@@ -229,9 +233,25 @@ describe('When executing an operation for the first time', () => {
 
       collection = client.db('auto-connect-change').collection('auto-connect');
       cs = collection.watch();
+
+      client.on('commandSucceeded', ev => {
```

Comment:
Same question for this file.  Why are we changing this file if we don't use the changes in tests?